packages/headless/src/api/service/insight/get-interface/get-interface-config-response.ts

```diff
@@ -38,9 +38,12 @@ interface Badge {
   color: string;
 }
 
+export type InsightFieldType = 'string' | 'date' | 'number' | 'multi';
+
 interface Detail {
   field: string;
```

Comment:
Does the type of `field` vary based on the field type, or is it always a string?

Reply:
The type is always a string, it's the field name. Given that, i'm not sure examples make sense.

Reply:
Does this change things for you?